File: app/src/main/java/com/dhsdevelopments/potato/clientapi/sendmessage/SendMessageRequest.java
package com.dhsdevelopments.potato.clientapi.sendmessage;

import com.google.gson.annotations.SerializedName;

public class SendMessageRequest
{
    @SerializedName( "text" )
    public String text;

    public SendMessageRequest() {
    }

    public SendMessageRequest( String text ) {
        this.text = text;
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/sendmessage/SendMessageRequest.kt
package com.dhsdevelopments.potato.clientapi.sendmessage

import com.google.gson.annotations.SerializedName

class SendMessageRequest {
    @SerializedName("text")
    lateinit var text: String

    constructor() {
    }

    constructor(text: String) {
        this.text = text
    }
}
